apps/vaporgui/VizWin.cpp

```diff
@@ -667,7 +667,7 @@ void VizWin::_renderHelper(bool fast)
     if (_getCurrentMouseMode() == MouseModeParams::GetRegionModeName()) {
         updateManip();
         if (_getRenderParams() && _getRenderParams()->GetOrientable()) { _updateOriginGlyph(); }
-        if (dynamic_cast<ContourParams *>(_getRenderParams())) _drawContourSliceQuad();
+        if (dynamic_cast<ContourParams *>(_getRenderParams()) || dynamic_cast<SliceParams *>(_getRenderParams())) _drawContourSliceQuad();
```

Comment:
You could just use isorientable now or just put it in the same if block.

Reply:
Note - I wasn't sure what you meant by "the same if block" but I think you're saying to put the _drawContourSliceQuad() call in the same block as _updateOriginGlyph().  Updated.